app.js

```diff
@@ -23,6 +23,7 @@ app.get("/api", getEndpoint);
 app.get("/api/topics", getAlltopics);
 app.get("/api/articles/:article_id", getArticleByID);
 app.get("/api/articles", getArticles);
+app.get("/api/articles?sort_by=:sort_by&order=:order", getArticles);
```

Comment:
You shouldn't need this line, the previous match (line 25) should catch the endpoint and handle the queries

Reply:
My instinct is this is already what is happening, if you delete line 26 here I think your tests will still pass